### ISSUE TITLE
Add requirements for packages and modules as we go, merge into main branch once project is complete or near complete.

### ISSUE THREAD

**Issue by Triggsy1091** [Triggsy1091/lang_translation#2]
Need to compile a list of requirements for this project to run i.e. google_trans_new, six, requests, etc. and push to main once complete.

**Comment by Triggsy1091** [comment]
Need to make these changes in the requirements branch not main
